scanomatic/data_processing/phases/segmentation.py

```diff
@@ -1,6 +1,6 @@
 import operator
 from enum import Enum
-from typing import Optional, Union
+from typing import Iterator, Optional, Union, cast
```

Comment:
Iterator, Sequence etc are deprecated in `typing` and should be imported from `collections.abc`. It's mostly `Optional`, `TypeVar`, `Union`, `Any` and `cast` that should come from `typing`.

Reply:
thanks for the info. fixed.